src/Reporter/Reporter.php

```diff
@@ -35,21 +35,21 @@ public function __construct()
 
     public function progress($current, $total)
     {
-        echo sprintf("Reviewing file %d of %d.\r", $current, $total);
+        echo sprintf("Reviewing %d of %d." . PHP_EOL, $current, $total);
```

Comment:
Ah, I see... that makes more sense with what I was seeing when running it. I will revert this change.